message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -21,8 +21,8 @@
     <enum name="MAV_CMD">
       <entry value="211" name="MAV_CMD_DO_GRIPPER">
         <description>Mission command to operate EPM gripper.</description>
-        <param index="1">Gripper number (a number from 1 to max number of grippers on the vehicle).</param>
-        <param index="2">Gripper action (0=release, 1=grab. See GRIPPER_ACTIONS enum).</param>
+        <param index="1" label="Number" minValue="1" increment="1">Gripper number (a number from 1 to max number of grippers on the vehicle).</param>
```

Comment:
Should be canonicalise on "instance" for this sort of thing rather than "number"?

Reply:
I really like that idea. @amilcarlucas can you make it happen?  We should also do in common.xml.